### PR TITLE
fix(storefront): BCTHEME-1780 Content that functions as a list should be indicated as such to screen reader users

### PR DESCRIPTION
**Pull request by bc-vlad-dlogush**
#### What?

This PR makes payment images as a list in order to meet WCAG 2.0 Level A

**Tickets / Documentation**

- [BCTHEME-1780](https://bigcommercecloud.atlassian.net/browse/BCTHEME-1780)

**Screenshots / Screen Recording**

https://github.com/bigcommerce/cornerstone/assets/83779098/ded51707-0fbe-4444-96f5-f7d3ac149ddd

https://github.com/bigcommerce/cornerstone/assets/83779098/174bfedc-17a1-4c93-bbb6-3251a366d73e

[BCTHEME-1780]: https://bigcommercecloud.atlassian.net/browse/BCTHEME-1780?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: assets/scss/layouts/footer/_footer.scss [hunk 1]
```diff
@@ -144,17 +144,29 @@
 }
 
 .footer-payment-icons {
+    display: flex;
+    flex-wrap: wrap;
     height: remCalc(40);
```
Comment (yurytut1993) [inline assets/scss/layouts/footer/_footer.scss:149]: looks like we should remove fixed height as now list items can be out of list when we have more than one line

File: assets/scss/layouts/footer/_footer.scss [hunk 2]
```diff
@@ -144,17 +144,29 @@
...
+    }
+
     @include breakpoint("medium") {
         margin-top: spacing("double");
     }
 
+    .footer-payment-item {
+        height: 100%;
+    }
+
```
Comment (yurytut1993) [inline assets/scss/layouts/footer/_footer.scss:166]: two questions about that:
1. why we need to set the height of `footer-payment-item`?
2. if we need - can we use `align-items: stretch;`for parent?

**Review by yurytut1993 — APPROVED** [review]
LGTM

File: templates/components/common/payment-icons.html [hunk 3]
```diff
@@ -1,30 +1,70 @@
...
+                <title>Klarna</title>
+                <use href="#icon-logo-klarna"></use>
+            </svg>
+        </li>
         {{/if}}
-    </div>
+    </ul>
     {{/or}}
-{{/with}}
+{{/with}}
```
Comment (bc-yevhenii-buliuk) [inline templates/components/common/payment-icons.html:70]: @bc-vlad-dlogush remove please an extra space

**Review by bc-yevhenii-buliuk — APPROVED** [review]

File: templates/components/common/payment-icons.html (hunk 3 above)
Comment (bc-vlad-dlogush) [reply templates/components/common/payment-icons.html]: Thanks, done!

File: templates/components/common/payment-icons.html [hunk 4]
```diff
@@ -1,30 +1,70 @@
...
+                <title>Amazonpay</title>
+                <use href="#icon-logo-amazonpay"></use>
+            </svg>
+        </li>
         {{/if}}
         {{#if show_accept_googlepay}}
-        <svg class="footer-payment-icon"><use href="#icon-logo-googlepay"></use></svg>
+        <li>
+            <svg class="footer-payment-icon" role="img">
+                <title>Googlepay</title>
```
Comment (yurytut1993) [inline templates/components/common/payment-icons.html:55]: I believe better change it to `Google Pay`

Comment (bc-vlad-dlogush) [reply templates/components/common/payment-icons.html]: Done!

Comment (yurytut1993) [reply templates/components/common/payment-icons.html]: @VladDlogush can you please update `Amazon Pay` also

Comment (bc-vlad-dlogush) [reply templates/components/common/payment-icons.html]: Now is better!

**Pull request merged**